src/main/java/frc/robot/superstructure/Superstructure.java

```diff
@@ -239,7 +265,8 @@ public Command transferElevatorShooter() {
 				new ParallelCommandGroup(
 					intakeStatesHandler.setState(IntakeStates.STOP),
 					funnelStateHandler.setState(FunnelState.STOP)
-				)
+				),
+				enableChangeStateAutomatically(true)
```

Comment:
I think it's better to put it inside the parallel of the funnel stop and intake stop